doc/openvas.8.in

```diff
@@ -133,6 +133,9 @@ This is the maximum lifetime, in seconds of a plugin. It may happen that some pl
 .IP scanner_plugins_timeout
 Like plugins_timeout, but for ACT_SCANNER plugins.
 
+.IP heartbeat_enabled
+Sometimes, a host is found alive at the beginning of the scan, but later the host is shut down, a firewall blocks the traffic, a network device issue break the connection, etc. This lead to many NVT timeouts and over long scan duration. This options allows to set a host as dead if the host does not respond anymore once the scan has started. The heartbeat check is trigger for each nvt timeout. This option accepts only integers. Integers greater than 0 (zero) are the maximum number of NVT timeouts before setting the host as dead. This option requiers Boreas alive test to be enabled. Options "unscanned_closed = no" and "unscanned_closed_udp = no" can produce NVT timeouts, and therefore you have to pay special atention in this case, as the host can be set as dead, when it is still alive. Default: option not set, disabled.
```

Comment:
```suggestion
During a scan it might happen that a host unexpectedly shuts down, a firewall blocks the traffic, a network device issue break the connection, etc. This leads to many NVT timeouts and long scan durations. This option checks the alive status of the host again after the provided amount of max NVT timeouts are reached. If the host is considered dead the scan will be stopped for this host. Otherwise the scan will continue. This option requires Boreas alive test to be enabled. Options "unscanned_closed = no" and "unscanned_closed_udp = no" can produce NVT timeouts, and therefore special attention has to be payed in those cases, as the host can be set as dead, when it is still alive. Default: option not set, disabled.
```